docs/lib/analytics/fragments/flutter/getting-started/20_installLib.md

```diff
@@ -6,13 +6,13 @@ Add Analytics by adding these libraries into your dependencies block:
 
 ```yaml 
 environment:
-  sdk: ">=2.11.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
question: Do we even need this `environment` block in these examples? My understanding is that we were previously including it because it was in the instructions for disabling sound null safety in your app and this gave added context, essentially repeating that. Because that is no longer the requirement, do we need to have anything related the sdk version? This seems to suggest folks should include this change, enabling sound null safety, which we are not explicitly saying.

Reply:
I don't think we need it but I do think it provides some context as to what a customer who might be unfamiliar with updating pubspec dependencies is looking at. OTOH, an argument could be made that a working knowledge of Flutter (by extension, pubspec.yaml) should be required before even starting with these docs. I don't have strong opinions on this.

Reply:
Fine explanation, fine as you have it here as I could also go either way. I think the flutter ecosystem push to null safety as default makes this part less likely to be confusing.